### PR TITLE
Exploring Unnecessary Conditionals: Mutually-Exclusive

### PR DESCRIPTION
**Pull request by SriHV**
### What is the context of this PR?
https://github.com/ONSdigital/design-system/issues/3025 .... Added new params that were missing in _macro_options.md

### How to review this PR
Check that params mentioned in examples are present in readme

### Checklist

This needs to be completed by the person raising the PR.

- [x] I have selected the correct Assignee
- [x] I have linked the correct Issue

### CODE REVIEW

**Review by rmccar — CHANGES_REQUESTED** [review]
`autosuggestResults` and the fieldset params are missing

**Review by rmccar — CHANGES_REQUESTED** [review]
So the way we have the params in other components are that the initial table is set up for params passed in with `param.<paramName>` then all the other tables like the fieldset one you have done here are for params passed in as an object to another one. Here I think they should all be in the same table. Check https://service-manual.ons.gov.uk/design-system/components/input#example.example-input-text.nunjucks-macro-options to see what I mean

File: src/components/mutually-exclusive/_macro-options.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+| legendClasses                    | string                               | false                                        | Classes to apply to the legend element                                                                                                                                                                     |
+| legendIsQuestionTitle            | boolean                              | true (unless `legend` is set)                | Creates an `h1` inside the `legend`. Use when there is only a single fieldset on the page                                                                                                                  |
+| legendClasses                    | string                               | false                                        | Classes to apply to the legend element                                                                                                                                                                     |
```
Comment (rmccar) [inline src/components/mutually-exclusive/_macro-options.md:8]: This one needs to be removed because you have it twice

Comment (SriHV) [reply src/components/mutually-exclusive/_macro-options.md:8]: oh sry might have copied it by mistake

**Review by rmccar — APPROVED** [review]

**Pull request merged**